sky/data/storage.py

```diff
@@ -921,17 +976,23 @@ def add_if_not_none(key: str, value: Optional[Any]):
         add_if_not_none('store', stores)
         add_if_not_none('persistent', self.persistent)
         add_if_not_none('mode', self.mode.value)
+        if self.mode == StorageMode.CSYNC:
+            add_if_not_none('interval_seconds', self.interval_seconds)
         if self.force_delete:
             config['_force_delete'] = True
         return config
 
+    def get_storage_name(self):
+        return self.name
+
 
 class S3Store(AbstractStore):
     """S3Store inherits from Storage Object and represents the backend
     for S3 buckets.
     """
 
     _ACCESS_DENIED_MESSAGE = 'Access Denied'
+    _CSYNC_DEFAULT_INTERVAL_SECONDS = 600
```

Comment:
This `600` also appeared in `sky_csync.py`. Can we use some constants? This is to decrease the possibility of later changes only applying to one of them.

Reply:
Placed the constant in `sky_csync.py` and imported the value to be used in `storage.py`